src/lib.rs

```diff
@@ -390,6 +390,21 @@ impl Device {
 impl MpsseCmdExecutor for Device {
     type Error = io::Error;
 
+    /// Initialize the MPSSE controller.
+    ///
+    /// According to AN135 [FTDI MPSSE basics], this method does the following:
+    /// 1. Optionally resets the peripheral side of FTDI port.
+    /// 2. Configures the maximum USB transfer sizes.
+    /// 3. Disables any event or error special characters.
+    /// 4. Configures the read and write timeouts.
```

Comment:
Hm, it does not seem to do that?..

Reply:
Right, we don't do it so far sticking to default ones. Unless I am missing something this setting is not directly available via libftdi (unlike libft2xx). I guess I have to mark it as `not implemented` or something like this.

Reply:
These settings are semi-public - the corresponding fields in `ftdi_context` are technically available, but that struct is documented as "Do not access directly if possible.".

Other than this I'm fine with this MR - please add the note, rebase fixups into the relevant commits and I'll merge it.

Reply:
Done